modules/ximgproc/test/test_globalmatting.cpp

```diff
@@ -0,0 +1,80 @@
+#include "test_precomp.hpp"
+using namespace cv;
+
+namespace opencv_test
+{
+namespace
+{
+const std::string INPUT_DIR       = "cv/ximgproc";
+const std::string IMAGE_FILENAME  = "input/doll.png";
+const std::string TRIMAP_FILENAME = "trimap/doll.png";
+
+
+class CV_GlobalMattingTest : public cvtest::BaseTest
+{
```

Comment:
This class is not needed (`BaseTest` is a legacy stuff).
Put all test code into `TEST()` body.

Reply:
Should I use TEST instead of BaseTest, sorry sir, I didn't quite get it.

Reply:
You should not use `cvtest::BaseTest` (it is legacy).

`TEST()` and other GoogleTest stuff is enough.